docker/docker_client_test.go

```diff
@@ -85,7 +85,8 @@ func TestDockerCertDir(t *testing.T) {
 			filepath.Join(variableReference, registryHostPort),
 		},
 	} {
-		path := dockerCertDir(c.ctx, registryHostPort)
+		path, err := dockerCertDir(c.ctx, registryHostPort)
+		assert.Equal(t, err, nil)
```

Comment:
`require.Equal` would more accurate; if `err` is non-`nil`, comparing the returned `path` value is meaningless.

Reply:
fixed